pages/ajax.render.php

```diff
@@ -378,7 +377,7 @@ function LogErrorMessage($sMsgPrefix, $aContextInfo) {
 					$oObj = null;
 				}
 				$oWidget = new UIExtKeyWidget($sTargetClass, $iInputId, '', $bSearchMode);
-				$oWidget->AutoComplete($oPage, $sFilter, $oObj, $sContains, $sOutputFormat, $sAutocompleteOperation);
+				$oWidget->AutoComplete($oPage, $sFilter, $oObj, $sContains, $sOutputFormat, '');
```

Comment:
Default value is not `''` but `null`. Anyway we might just not pass this argument if it is not used, what do you think?

```suggestion
				$oWidget->AutoComplete($oPage, $sFilter, $oObj, $sContains, $sOutputFormat);
```